tests/notebooks/basic_relative.ipynb

```diff
@@ -0,0 +1,53 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# a title\n",
+    "\n",
+    "some text\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test reading a relative file to make sure relative paths work\n",
+    "with open(\"./conf.py\") as ff:\n",
```

Comment:
@choldgraf I guess this PR is missing the `conf.py` file?

Reply:
no it shows up when the test is run - I think the fixture builds a little sphinx site in a tmp folder, so `conf.py` is there

Reply:
@choldgraf ooh cool. gotcha.

Reply:
@choldgraf but it is showing the error of file not found for the test in the local system.

sorry for the unclear screenshot. I had put a debugger in `test_relative_path_cache`

![image](https://user-images.githubusercontent.com/6542997/84581980-6cc58e80-ae29-11ea-815f-a6b2ddfc9fac.png)

So, the execution failed, and when I saw the log file mentioned in the screenshot. It said `conf.py` file not found. Can you verify it in your system?

Reply:
hmmm, that is weird - I just tried it and it worked for me. are you on latest myst-nb/jupyter-cache?

Reply:
Hmm. this is strange. Like for example, I created the PR, in which I changed the assert statement to `Execution Succeeded`. https://github.com/executablebooks/MyST-NB/pull/199 . And it is failing, similar to in my local system.